pkg/csi_driver/node.go

```diff
@@ -79,16 +80,31 @@ func (s *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublish
 	// Mount source
 	source := fmt.Sprintf("%s:/%s", attr[attrIp], attr[attrVolume])
 
+	// FileSystem type
+	fstype := "nfs"
+
 	// Mount options
 	options := []string{}
+
+	// Windows specific values
+	if runtime.GOOS == "windows" {
+		source = fmt.Sprintf("\\\\%s\\%s", attr[attrIp], attr[attrVolume])
+		fstype = "cifs"
```

Comment:
should we validate fstype in `validateVolumeCapabilities`?

Reply:
The [mount utility](https://github.com/kubernetes/kubernetes/blob/5b496fe8f5933de9b53e35780a486c8f7736d02b/pkg/util/mount/mount_windows.go#L85) for Windows only supports CIFS so I think it's okay to leave this hardcoded for now.

Reply:
Correct, but the fstype is set by the user and passed to us.  So the point of `validateVolumeCapabilities` is to make sure we're able to support what the user requested.

Reply:
Done?